public/styles/component.scss

```diff
@@ -118,4 +118,8 @@ kbn-dis doc-table .kbnDocViewer__warning {
 }
 .header__breadcrumbsWithExtensionContainer .header__breadcrumbsAppendExtension {
     flex-grow: 0;
+}
+
+.euiCard .euiCard__top .euiIcon--app .euiIcon__fillSecondary{
+    fill: #256BD1
```

Comment:
**thought**: This style could be affecting some components of Kibana apart from the Wazuh app. We should restrict this to the Wazuh app.

Reply:
**thought**: Was the secondary color changed? In [this comment](https://github.com/wazuh/wazuh-kibana-app/issues/3575#issuecomment-951966154), the color seems to be `#3585F9`. Could we be sure that color we should apply?